src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksNewVendorPayeeAchReportServiceImpl.java

```diff
@@ -246,6 +277,78 @@ public String getUnprocessedSubTitle() {
         return unprocessedSubTitle;
     }
 
+    public String getKfsVendorNumberLabel() {
+        if (ObjectUtils.isNull(kfsVendorNumberLabel)) {
+            setKfsVendorNumberLabel(getPaymentWorksBatchUtilityService().retrievePaymentWorksParameterValue(
+                    PaymentWorksParameterConstants.PAYMENTWORKS_PAYEE_ACH_REPORT_KFS_VENDOR_NUMBER_LABEL));
```

Comment:
This parameter name would need to be changed to PAYMENTWORKS_NEW_VENDOR_REPORT_KFS_VENDOR_NUMBER_LABEL if the attribute changes classes.

Reasoning for the naming convention: PaymentWorks looks at vendors as being either "new" or "edited" where the data returned by the web service calls is different. There are three batch jobs and a user's manual action that all must work together in order to process a single "new" PMW vendor into KFS before that vendor can be uploaded back to PMW with the KFS identifiers. The batch reports data, services, and implementation classes were designed around this "new" vs "edited" process to take advantage of reuse on our side. So, high level items that could be used in any of the three batch jobs would be in the base classes and their parameters would begin with PAYMENTWORKS_NEW_VENDOR_REPORT_... while items that are needed by a specific report would have that specific report's identifier, i.e. PAYMENTWORKS_PAYEE_ACH_REPORT_...   In addition, items that are common between the reports but need to be unique to each report, i.e. file names, email addresses, labels, etc., were made abstract in the base class and specified in each implementation.

Reply:
Ok, I'll rename this parameter accordingly.